heroku/cli
# refactor: migrate orgs/index to oclif/core

## Testing
- Check out branch and run `yarn` and `yarn build`
- Run `./bin/run orgs`
    - You should get a table with two columns, one for Teams and one for Roles.
    - This will be slightly different from the `heroku orgs` command in that it has table headers.
- Run `./bin/run orgs --json`
    - You should get json output
